README.rst

```diff
@@ -28,41 +28,16 @@ not yet available.
 Installation
 ============
 
-Download the zip file and unzip it; or, much better,
```

Comment:
@wesleybowman I find that explaining how to install from source is tedious and most users who are interested in doing that either know how to do it or will look for that information somewhere else.

Reply:
I was going to agree, but we did just have an issue where this was relevant (and where you told them handy info https://github.com/wesleybowman/UTide/issues/44#issuecomment-332603648). 

I am fine with removing this, just something to keep in mind. Maybe we could at least put 
```
pip install git+https://github.com/wesleybowman/UTide.git
```
? It's less overall instruction, but would allow users to have the info at a passing glance.

Reply:
I don't agree that info should be there in the README, such info belongs in a development section of docs.

The reason for that is, IMO, when users need to `pip install` from the repo is b/c the devs failed to release often :smile: 
Also, only advanced users that want to change the code should do that. When we put those instructions in the README we encourage people to copy-n-paste without really knowing what they are getting, even if it is clearly explained in the text that it is a "develop install."  Most users don't really know that means and the chance of getting an unstable code increases.

Reply:
Alright, good points were made here :+1: